index.js

```diff
@@ -44,7 +44,7 @@ types[VISA] = {
   prefixPattern: /^4$/,
   exactPattern: /^4\d*$/,
   gaps: [4, 8, 12],
```

Comment:
We don't actually know how visa is going to format the numbers on their cards yet. So a 19 digit visa card looks like this on braintree-web:

![screen shot 2017-02-06 at 2 31 09 pm](https://cloud.githubusercontent.com/assets/2916945/22665069/ff3cabac-ec78-11e6-9122-62fecdef067d.png)